code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -30,7 +30,9 @@
 	var/assembled = 0
 	active = 1
 	det_time = 50
-
+	var/list/shrapnel_list = new()
```

Comment:
It's generally better to start with the var holding the list as `null` and then initialize the list when it's needed. Not a big deal here because it's on a relatively uncommon item, but if you ever add a list to something more common, keep that in mind.